### ISSUE TITLE
Can this plugin save the user object to the installation object?

### ISSUE THREAD

**Issue by gespim** [avivais/phonegap-parse-plugin#46]
I need the Installation object  has user attribute so that we can send push notification to a given set of users.

**Comment by castillobg** [comment]
Maybe you can try using the getInstallationId method?

**Comment by gespim** [comment]
No. I need to store the user id into Installation object of Parse.  It is needed for later pushing to specific target user group.   Could you guys update the plugin to make it possible?

**Comment by gespim** [comment]
I suggest the author add this function to be exposed to plugin user to call the userassignment to the installation object in parse

void ParsePushUserAssign(void)
//-------------------------------------------------------------------------------------------------------------------------------------------------
{
	PFInstallation *installation = [PFInstallation currentInstallation];
	installation[PF_INSTALLATION_USER] = [PFUser currentUser];
	[installation saveInBackgroundWithBlock:^(BOOL succeeded, NSError *error)
	{
		if (error != nil)
		{
			NSLog(@"ParsePushUserAssign save error.");
		}
	}];
}
